sql/core/src/test/scala/org/apache/spark/sql/DatasetBenchmark.scala

```diff
@@ -43,7 +43,7 @@ object DatasetBenchmark {
       var res = rdd
       var i = 0
       while (i < numChains) {
-        res = rdd.map(func)
```

Comment:
Here, obviously, this should be `res = res.map(func) `.
The original code applies map only once to the input `rdd` regardless of `numChains`.